components/system/modules/FriendsList.js

```diff
@@ -0,0 +1,245 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+import * as Strings from "~/common/strings";
+import * as SVG from "~/components/system/svg";
+
+import { css } from "@emotion/react";
+import { Table } from "~/components/system/components/Table";
+import { StatUpload, StatDownload } from "~/components/system/components/Stat";
+
+let genericImg =
+  "https://hub.textile.io/ipfs/bafybeiblly23jomdjjiq7ilth667npcfm5llqb5xfstodbbfa5pxtoek7u";
+
+const STYLES_BUTTON = `
+  font-family: ${Constants.font.text};
+  font-size: ${Constants.typescale.lvl1};
+  cursor: pointer;
+  background: none;
+  border: none;
+  outline: none;
+  padding: 8px;
+
+  :hover {
+    background-color: ${Constants.system.gray};
+    border-radius: 4px;
+  }
+`;
+
+const STYLES_ACCEPT_BUTTON = css`
+  ${STYLES_BUTTON}
+  color: ${Constants.system.brand};
+`;
+
+const STYLES_REJECT_BUTTON = css`
+  ${STYLES_BUTTON}
+  color: ${Constants.system.black};
+`;
+
+const centerLeftStyle = {
+  display: "flex",
+  height: "100%",
+  alignItems: "center",
+};
+
+const centerRightStyle = {
+  display: "flex",
+  height: "100%",
+  alignItems: "center",
+  justifyContent: "flex-end",
+};
+
+export class FriendsList extends React.Component {
+  state = {
+    selectedRowId: null,
+  };
+
+  _handleClick = (e) => {
+    this.setState({ selectedRowId: e.target.value });
+  };
+
+  render() {
+    return (
+      <React.Fragment>
+        <div
+          style={{ fontSize: Constants.typescale.lvl2, marginBottom: "8px" }}
```

Comment:
Just make these classes? It will be faster than computing inline styles performance wise.